### DISCUSSION TITLE
Please add an ability to publish draft releases from mobile

### DISCUSSION THREAD

**Discussion by risenxxx** [community/community#75991] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.138.0 (115117336)
Device: iPhone 12 mini (iOS 17.1.1)
</details>

**Comment by kdinthehouse** [comment]
No, there is currently no ability to publish draft releases from the GitHub mobile app. You can only create and save draft releases, and then you will need to publish them from a desktop computer.

To create and save a draft release on the GitHub mobile app, follow these steps:

Open the GitHub mobile app and go to the repository you want to create a release for.
Tap on the "Releases" tab.
Tap on the "+" button.
Enter the release name, tag, and body.
Tap on the "Save draft" button.
The draft release will be saved to the repository, but it will not be published until you publish it from a desktop computer. To publish a draft release from a desktop computer, follow these steps:

Open a web browser and go to the GitHub repository you want to publish the release for.
Click on the "Releases" tab.
Click on the draft release you want to publish.
Click on the "Publish release" button.
The release will be published to the repository.
